Rock/Workflow/Action/Communications/SendNotification.cs

```diff
@@ -42,6 +42,7 @@ namespace Rock.Workflow.Action
     [WorkflowAttribute("Sound", "The choice of sound or an attribute that contains the choice of sound that should be sent.", false, "True", "", 2, "Sound", new string[] { "Rock.Field.Types.BooleanFieldType" })]
     [WorkflowTextOrAttribute("Message", "Attribute Value", "The message or an attribute that contains the message that should be sent. <span class='tip tip-lava'></span>", true, "", "", 3, "Message",
         new string[] { "Rock.Field.Types.TextFieldType" })]
+    [WorkflowTextOrAttribute("Url", "Attribute Value", "The URL or an attribute that contains the URL that the notification should link to.", false, "", "", 4, "Url", new string[] { "Rock.Field.Types.TextFieldType" })]
```

Comment:
@richarddubay should new string[] { "Rock.Field.Types.TextFieldType" } also include Rock.Field.Types.UrlLinkFieldType?

Reply:
I think that's a good call out but I'm not really sure if it's needed here or not. I see where that is used in `CreateShortLink` for sure. Would you add it?